presto-main/src/main/java/com/facebook/presto/util/GraphvizPrinter.java

```diff
@@ -464,7 +472,7 @@ public Void visitLimit(LimitNode node, Void context)
         @Override
         public Void visitTableScan(TableScanNode node, Void context)
         {
-            printNode(node, format("TableScan[%s]", node.getTable()), NODE_COLORS.get(NodeType.TABLESCAN));
+            printNode(node, format("TableScan | [%s]", node.getTable()), NODE_COLORS.get(NodeType.TABLESCAN));
```

Comment:
Thanks for fixing this `TableScan` label formatting. Usually, `TableHandles` are long strings, so having this formatted with `|` will move these to the next line, improving readability